.gitignore

```diff
@@ -3,3 +3,4 @@
 /memprofile.out
 /dist/
 /.vscode/
+.idea
```

Comment:
Should I be adding this here (it makes sense to add it if it's just our repo)? How likely are we merging this upstream?

Reply:
Think this is fine